aio_pika/robust_connection.py

```diff
@@ -95,7 +95,9 @@ async def _on_connected(self) -> None:
             raise RuntimeError("No active transport for connection %r", self)
 
         try:
-            for channel in self.__channels:
+            # Make a copy of the channels to iterate on to guard from
+            # concurrent updates to the set.
+            for channel in list(self.__channels):
```

Comment:
It would be better to use a tuple instead of a list due to execution speed and lesser memory consumption.